doozerlib/cli/release_gen_payload.py

```diff
@@ -1226,7 +1241,7 @@ def build_payload_istag(payload_tag_name: str, payload_entry: PayloadEntry) -> D
         }
 
     @staticmethod
-    def build_payload_imagestream(imagestream_name: str, imagestream_namespace: str, payload_istags: Iterable[Dict], assembly_wide_inconsistencies: Iterable[AssemblyIssue]) -> Dict:
+    def build_payload_imagestream(runtime, imagestream_name: str, imagestream_namespace: str, payload_istags: Iterable[Dict], assembly_wide_inconsistencies: Iterable[AssemblyIssue]) -> Dict:
```

Comment:
Docstring should include `runtime` param description

Reply:
Added